frontend/packages/dev-console/src/components/topology/application-panel/__tests__/ApplicationGroupResource.spec.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { Link } from 'react-router-dom';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+import TopologyApplicationResourceList from '../TopologyApplicationList';
+import ApplicationGroupResource from '../ApplicationGroupResource';
+import { sampleDeployments } from '../../__tests__/topology-test-data';
+
+describe(ApplicationGroupResource.displayName, () => {
+  it('should component exists', () => {
+    const wrapper = shallow(
+      <ApplicationGroupResource
+        title="Deployments"
+        resourcesData={sampleDeployments.data}
+        group="a"
+      />,
+    );
+    expect(wrapper.exists()).toBe(true);
```

Comment:
https://github.com/openshift/console/pull/3851#discussion_r363235342

Use `.isEmptyRender()` equal to false to showcase it didn't render an empty husk.

Reply:
fixed